src/data/fintype/basic.lean

```diff
@@ -961,6 +964,14 @@ instance finset.fintype [fintype α] : fintype (finset α) :=
   fintype.card (finset α) = 2 ^ (fintype.card α) :=
 finset.card_powerset finset.univ
 
+@[simp] lemma fintype.card_finset_len [fintype α] (k : ℕ) :
+  fintype.card {s : finset α // s.card = k} = nat.choose (fintype.card α) k :=
```

Comment:
Is the LHS the canonical way to talk about the number of elements satisfying a property? I would expect it to be `(finset.univ.filter ...).card`